### PR TITLE
added loriot lora provider + updated firefly provider to provide bandwidth

### PR DESCRIPTION
**Pull request by BobClaerhout**

### CODE REVIEW

File: adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/providers/LoriotProvider.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 Contributors to the Eclipse Foundation
```
Comment (ctron) [inline adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/providers/LoriotProvider.java]:
```suggestion
 * Copyright (c) 2019 Contributors to the Eclipse Foundation
```

File: adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/LoriotProviderTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 Contributors to the Eclipse Foundation
```
Comment (ctron) [inline adapters/lora-vertx/src/test/java/org/eclipse/hono/adapter/lora/providers/LoriotProviderTest.java]:
```suggestion
 * Copyright (c) 2019 Contributors to the Eclipse Foundation
```

**Comment by BobClaerhout** [comment]
> Looks good to me. Only a small header issue.

Done

**Comment by sophokles73** [comment]
@BobClaerhout you have created the PR against the 1.0.x branch. However, that branch is for bugfixes only. Please re-create the PR against master, which is the branch we are using for new features towards 1.1.0.

**Comment by BobClaerhout** [comment]
Closing in favor of https://github.com/eclipse/hono/pull/1624

**Pull request closed without merging**